### ISSUE TITLE
firefly-iii:upgrade-database does not work in 6.0.7

### ISSUE THREAD

**Issue by gramakri** [firefly-iii/firefly-iii#7349]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Starting with 6.0.7, the `php artisan firefly-iii:upgrade-database` errors.

### Debug information

```
# php artisan firefly-iii:upgrade-database
Now seeding the database...
[2023-04-09 17:07:14] local.WARNING: Database entry exists. Update to latest version (19)  

   INFO  Nothing to migrate.  

   INFO  Seeding database.  

  Database\Seeders\AccountTypeSeeder ....................................................................................................... RUNNING  
  Database\Seeders\AccountTypeSeeder ................................................................................................. 21.06 ms DONE  

  Database\Seeders\TransactionCurrencySeeder ............................................................................................... RUNNING  
  Database\Seeders\TransactionCurrencySeeder ......................................................................................... 24.54 ms DONE  

  Database\Seeders\TransactionTypeSeeder ................................................................................................... RUNNING  
  Database\Seeders\TransactionTypeSeeder .............................................................................................. 6.94 ms DONE  

  Database\Seeders\PermissionSeeder ........................................................................................................ RUNNING  
  Database\Seeders\PermissionSeeder ................................................................................................... 2.20 ms DONE  

  Database\Seeders\LinkTypeSeeder .......................................................................................................... RUNNING  
  Database\Seeders\LinkTypeSeeder ..................................................................................................... 4.17 ms DONE  

  Database\Seeders\ConfigSeeder ............................................................................................................ RUNNING  
  Database\Seeders\ConfigSeeder ....................................................................................................... 6.83 ms DONE  

  Database\Seeders\UserRoleSeeder .......................................................................................................... RUNNING  
  Database\Seeders\UserRoleSeeder ..................................................................................................... 8.36 ms DONE  

  Database\Seeders\ExchangeRateSeeder ...................................................................................................... RUNNING  
  Database\Seeders\ExchangeRateSeeder ................................................................................................. 2.18 ms DONE  

Fix PostgreSQL sequences.
[2023-04-09 17:07:14] local.ERROR: Exception is: {"class":"Illuminate\\Database\\QueryException","errorMessage":"SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"export_jobs\" does not exist\nLINE 1: select MAX(id) from \"export_jobs\" limit 1\n                            ^ (Connection: pgsql, SQL: select MAX(id) from \"export_jobs\" limit 1)","time":"Sun, 09 Apr 2023 17:07:14 +0200","file":"\/app\/code\/vendor\/laravel\/framework\/src\/Illuminate\/Database\/Connection.php","line":793,"code":"42P01","version":"6.0.7","url":"https:\/\/ff.smartserver.io","userAgent":"Symfony","json":true,"method":"GET"}  
[2023-04-09 17:07:14] local.WARNING: [Bcc] Could not email or log the error. Please validate your email settings, use the .env.example file as a guide.  
[2023-04-09 17:07:14] local.ERROR: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "export_jobs" does not exist
LINE 1: select MAX(id) from "export_jobs" limit 1
                            ^ (Connection: pgsql, SQL: select MAX(id) from "export_jobs" limit 1) {"exception":"[object] (Illuminate\\Database\\QueryException(code: 42P01): SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"export_jobs\" does not exist
LINE 1: select MAX(id) from \"export_jobs\" limit 1
                            ^ (Connection: pgsql, SQL: select MAX(id) from \"export_jobs\" limit 1) at /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php:793)
[stacktrace]
#0 /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php(753): Illuminate\\Database\\Connection->runQueryCallback()
#1 /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php(405): Illuminate\\Database\\Connection->run()
#2 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2718): Illuminate\\Database\\Connection->select()
#3 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2707): Illuminate\\Database\\Query\\Builder->runSelect()
#4 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(3248): Illuminate\\Database\\Query\\Builder->Illuminate\\Database\\Query\\{closure}()
#5 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2706): Illuminate\\Database\\Query\\Builder->onceWithColumns()
#6 /app/code/vendor/laravel/framework/src/Illuminate/Database/Concerns/BuildsQueries.php(296): Illuminate\\Database\\Query\\Builder->get()
#7 /app/code/app/Console/Commands/Correction/FixPostgresSequences.php(125): Illuminate\\Database\\Query\\Builder->first()
#8 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): FireflyIII\\Console\\Commands\\Correction\\FixPostgresSequences->handle()
#9 /app/code/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#10 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#11 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\\Container\\BoundMethod::callBoundMethod()
#12 /app/code/vendor/laravel/framework/src/Illuminate/Container/Container.php(662): Illuminate\\Container\\BoundMethod::call()
#13 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(194): Illuminate\\Container\\Container->call()
#14 /app/code/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
#15 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(163): Symfony\\Component\\Console\\Command\\Command->run()
#16 /app/code/vendor/symfony/console/Application.php(1040): Illuminate\\Console\\Command->run()
#17 /app/code/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
#18 /app/code/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
#19 /app/code/vendor/laravel/framework/src/Illuminate/Console/Application.php(162): Symfony\\Component\\Console\\Application->run()
#20 /app/code/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(381): Illuminate\\Console\\Application->call()
#21 /app/code/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(353): Illuminate\\Foundation\\Console\\Kernel->call()
#22 /app/code/app/Console/Commands/Upgrade/UpgradeDatabase.php(140): Illuminate\\Support\\Facades\\Facade::__callStatic()
#23 /app/code/app/Console/Commands/Upgrade/UpgradeDatabase.php(58): FireflyIII\\Console\\Commands\\Upgrade\\UpgradeDatabase->callInitialCommands()
#24 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): FireflyIII\\Console\\Commands\\Upgrade\\UpgradeDatabase->handle()
#25 /app/code/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#26 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#27 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\\Container\\BoundMethod::callBoundMethod()
#28 /app/code/vendor/laravel/framework/src/Illuminate/Container/Container.php(662): Illuminate\\Container\\BoundMethod::call()
#29 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(194): Illuminate\\Container\\Container->call()
#30 /app/code/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
#31 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(163): Symfony\\Component\\Console\\Command\\Command->run()
#32 /app/code/vendor/symfony/console/Application.php(1040): Illuminate\\Console\\Command->run()
#33 /app/code/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
#34 /app/code/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
#35 /app/code/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(200): Symfony\\Component\\Console\\Application->run()
#36 /app/code/artisan(35): Illuminate\\Foundation\\Console\\Kernel->handle()
#37 {main}

[previous exception] [object] (PDOException(code: 42P01): SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"export_jobs\" does not exist
LINE 1: select MAX(id) from \"export_jobs\" limit 1
                            ^ at /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php:419)
[stacktrace]
#0 /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php(419): PDOStatement->execute()
#1 /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php(786): Illuminate\\Database\\Connection->Illuminate\\Database\\{closure}()
#2 /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php(753): Illuminate\\Database\\Connection->runQueryCallback()
#3 /app/code/vendor/laravel/framework/src/Illuminate/Database/Connection.php(405): Illuminate\\Database\\Connection->run()
#4 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2718): Illuminate\\Database\\Connection->select()
#5 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2707): Illuminate\\Database\\Query\\Builder->runSelect()
#6 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(3248): Illuminate\\Database\\Query\\Builder->Illuminate\\Database\\Query\\{closure}()
#7 /app/code/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2706): Illuminate\\Database\\Query\\Builder->onceWithColumns()
#8 /app/code/vendor/laravel/framework/src/Illuminate/Database/Concerns/BuildsQueries.php(296): Illuminate\\Database\\Query\\Builder->get()
#9 /app/code/app/Console/Commands/Correction/FixPostgresSequences.php(125): Illuminate\\Database\\Query\\Builder->first()
#10 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): FireflyIII\\Console\\Commands\\Correction\\FixPostgresSequences->handle()
#11 /app/code/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#12 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#13 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\\Container\\BoundMethod::callBoundMethod()
#14 /app/code/vendor/laravel/framework/src/Illuminate/Container/Container.php(662): Illuminate\\Container\\BoundMethod::call()
#15 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(194): Illuminate\\Container\\Container->call()
#16 /app/code/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
#17 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(163): Symfony\\Component\\Console\\Command\\Command->run()
#18 /app/code/vendor/symfony/console/Application.php(1040): Illuminate\\Console\\Command->run()
#19 /app/code/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
#20 /app/code/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
#21 /app/code/vendor/laravel/framework/src/Illuminate/Console/Application.php(162): Symfony\\Component\\Console\\Application->run()
#22 /app/code/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(381): Illuminate\\Console\\Application->call()
#23 /app/code/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(353): Illuminate\\Foundation\\Console\\Kernel->call()
#24 /app/code/app/Console/Commands/Upgrade/UpgradeDatabase.php(140): Illuminate\\Support\\Facades\\Facade::__callStatic()
#25 /app/code/app/Console/Commands/Upgrade/UpgradeDatabase.php(58): FireflyIII\\Console\\Commands\\Upgrade\\UpgradeDatabase->callInitialCommands()
#26 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): FireflyIII\\Console\\Commands\\Upgrade\\UpgradeDatabase->handle()
#27 /app/code/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#28 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#29 /app/code/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\\Container\\BoundMethod::callBoundMethod()
#30 /app/code/vendor/laravel/framework/src/Illuminate/Container/Container.php(662): Illuminate\\Container\\BoundMethod::call()
#31 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(194): Illuminate\\Container\\Container->call()
#32 /app/code/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
#33 /app/code/vendor/laravel/framework/src/Illuminate/Console/Command.php(163): Symfony\\Component\\Console\\Command\\Command->run()
#34 /app/code/vendor/symfony/console/Application.php(1040): Illuminate\\Console\\Command->run()
#35 /app/code/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
#36 /app/code/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
#37 /app/code/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(200): Symfony\\Component\\Console\\Application->run()
#38 /app/code/artisan(35): Illuminate\\Foundation\\Console\\Kernel->handle()
#39 {main}
"} 

   Illuminate\Database\QueryException 

  SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "export_jobs" does not exist
LINE 1: select MAX(id) from "export_jobs" limit 1
                            ^ (Connection: pgsql, SQL: select MAX(id) from "export_jobs" limit 1)

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:793
    789▕         // If an exception occurs when attempting to run a query, we'll format the error
    790▕         // message to include the bindings with SQL, which will make this exception a
    791▕         // lot more helpful to the developer instead of just the database's errors.
    792▕         catch (Exception $e) {
  ➜ 793▕             throw new QueryException(
    794▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    795▕             );
    796▕         }
    797▕     }

      +10 vendor frames 

  11  app/Console/Commands/Correction/FixPostgresSequences.php:125
      Illuminate\Database\Query\Builder::first()
      +14 vendor frames 

  26  app/Console/Commands/Upgrade/UpgradeDatabase.php:140
      Illuminate\Support\Facades\Facade::__callStatic()
```

### Expected behaviour

The command used to work fine will 6.0.6

### Steps to reproduce

I found this when updating the Cloudron package to 6.0.7 . But here is the list of steps which follow installation from source as per https://docs.firefly-iii.org/firefly-iii/installation/self_hosted/#initialize-the-database

1. Start with fresh firefly-III
2. Fix up env file
3. php artisan migrate:refresh --seed <-- this is OK. this runs the migrations
4. php artisan firefly-iii:upgrade-database <-- this causes the above error

### Additional info

I tried debugging this  by inspecting your Dockerfile at https://dev.azure.com/Firefly-III/_git/MainImage?path=/entrypoint.sh . But it doesn't use this command at all. Curiously, `app/Console/Commands/Correction/FixPostgresSequences.php` has no change as well between releases.

**Comment by gramakri** [comment]
Commenting out the tables (which were never created) in `app/Console/Commands/Correction/FixPostgresSequences.php` makes the command work.

```
            'budgets',
            'categories',
            'category_transaction',
            'category_transaction_journal',
            'configuration',
            'currency_exchange_rates',
//            'export_jobs',
            'failed_jobs',
            'group_journals',
//            'import_jobs',
            'jobs',
            'journal_links',
            'journal_meta',
            'limit_repetitions',
            'link_types',
            'locations',
```

Then:
```
# php artisan firefly-iii:upgrade-database
Now seeding the database...
[2023-04-09 17:33:33] local.WARNING: Database entry exists. Update to latest version (19)  

   INFO  Nothing to migrate.  

   INFO  Seeding database.  

  Database\Seeders\AccountTypeSeeder ....................................................................................................... RUNNING  
  Database\Seeders\AccountTypeSeeder ................................................................................................. 17.39 ms DONE  

  Database\Seeders\TransactionCurrencySeeder ............................................................................................... RUNNING  
  Database\Seeders\TransactionCurrencySeeder ......................................................................................... 20.61 ms DONE  

  Database\Seeders\TransactionTypeSeeder ................................................................................................... RUNNING  
  Database\Seeders\TransactionTypeSeeder .............................................................................................. 8.19 ms DONE  

  Database\Seeders\PermissionSeeder ........................................................................................................ RUNNING  
  Database\Seeders\PermissionSeeder ................................................................................................... 3.51 ms DONE  

  Database\Seeders\LinkTypeSeeder .......................................................................................................... RUNNING  
  Database\Seeders\LinkTypeSeeder ..................................................................................................... 3.98 ms DONE  

  Database\Seeders\ConfigSeeder ............................................................................................................ RUNNING  
  Database\Seeders\ConfigSeeder ....................................................................................................... 6.68 ms DONE  

  Database\Seeders\UserRoleSeeder .......................................................................................................... RUNNING  
  Database\Seeders\UserRoleSeeder .................................................................................................... 11.12 ms DONE  

  Database\Seeders\ExchangeRateSeeder ...................................................................................................... RUNNING  
  Database\Seeders\ExchangeRateSeeder ................................................................................................. 2.12 ms DONE  

Fix PostgreSQL sequences.
Going to verify PostgreSQL table sequences.
Checking the next id sequence for table "2fa_tokens".
nextval is NULL for table "2fa_tokens", go to next table.
Checking the next id sequence for table "account_meta".
nextval is NULL for table "account_meta", go to next table.
Checking the next id sequence for table "account_types".
Table "account_types" autoincrement is correct.
Checking the next id sequence for table "accounts".
nextval is NULL for table "accounts", go to next table.
Checking the next id sequence for table "attachments".
nextval is NULL for table "attachments", go to next table.
Checking the next id sequence for table "auto_budgets".
nextval is NULL for table "auto_budgets", go to next table.
Checking the next id sequence for table "available_budgets".
nextval is NULL for table "available_budgets", go to next table.
Checking the next id sequence for table "bills".
nextval is NULL for table "bills", go to next table.
Checking the next id sequence for table "budget_limits".
nextval is NULL for table "budget_limits", go to next table.
Checking the next id sequence for table "budget_transaction".
nextval is NULL for table "budget_transaction", go to next table.
Checking the next id sequence for table "budget_transaction_journal".
nextval is NULL for table "budget_transaction_journal", go to next table.
Checking the next id sequence for table "budgets".
nextval is NULL for table "budgets", go to next table.
Checking the next id sequence for table "categories".
nextval is NULL for table "categories", go to next table.
Checking the next id sequence for table "category_transaction".
nextval is NULL for table "category_transaction", go to next table.
Checking the next id sequence for table "category_transaction_journal".
nextval is NULL for table "category_transaction_journal", go to next table.
Checking the next id sequence for table "configuration".
Table "configuration" autoincrement is correct.
Checking the next id sequence for table "currency_exchange_rates".
nextval is NULL for table "currency_exchange_rates", go to next table.
Checking the next id sequence for table "failed_jobs".
nextval is NULL for table "failed_jobs", go to next table.
Checking the next id sequence for table "group_journals".
nextval is NULL for table "group_journals", go to next table.
Checking the next id sequence for table "jobs".
nextval is NULL for table "jobs", go to next table.
Checking the next id sequence for table "journal_links".
nextval is NULL for table "journal_links", go to next table.
Checking the next id sequence for table "journal_meta".
nextval is NULL for table "journal_meta", go to next table.
Checking the next id sequence for table "limit_repetitions".
nextval is NULL for table "limit_repetitions", go to next table.
Checking the next id sequence for table "link_types".
Table "link_types" autoincrement is correct.
Checking the next id sequence for table "locations".
nextval is NULL for table "locations", go to next table.
Checking the next id sequence for table "migrations".
Table "migrations" autoincrement is correct.
Checking the next id sequence for table "notes".
nextval is NULL for table "notes", go to next table.
Checking the next id sequence for table "oauth_clients".
nextval is NULL for table "oauth_clients", go to next table.
Checking the next id sequence for table "oauth_personal_access_clients".
nextval is NULL for table "oauth_personal_access_clients", go to next table.
Checking the next id sequence for table "object_groups".
nextval is NULL for table "object_groups", go to next table.
Checking the next id sequence for table "permissions".
nextval is NULL for table "permissions", go to next table.
Checking the next id sequence for table "piggy_bank_events".
nextval is NULL for table "piggy_bank_events", go to next table.
Checking the next id sequence for table "piggy_bank_repetitions".
nextval is NULL for table "piggy_bank_repetitions", go to next table.
Checking the next id sequence for table "piggy_banks".
nextval is NULL for table "piggy_banks", go to next table.
Checking the next id sequence for table "preferences".
nextval is NULL for table "preferences", go to next table.
Checking the next id sequence for table "recurrences".
nextval is NULL for table "recurrences", go to next table.
Checking the next id sequence for table "recurrences_meta".
nextval is NULL for table "recurrences_meta", go to next table.
Checking the next id sequence for table "recurrences_repetitions".
nextval is NULL for table "recurrences_repetitions", go to next table.
Checking the next id sequence for table "recurrences_transactions".
nextval is NULL for table "recurrences_transactions", go to next table.
Checking the next id sequence for table "roles".
Table "roles" autoincrement is correct.
Checking the next id sequence for table "rt_meta".
nextval is NULL for table "rt_meta", go to next table.
Checking the next id sequence for table "rule_actions".
nextval is NULL for table "rule_actions", go to next table.
Checking the next id sequence for table "rule_groups".
nextval is NULL for table "rule_groups", go to next table.
Checking the next id sequence for table "rule_triggers".
nextval is NULL for table "rule_triggers", go to next table.
Checking the next id sequence for table "rules".
nextval is NULL for table "rules", go to next table.
Checking the next id sequence for table "tag_transaction_journal".
nextval is NULL for table "tag_transaction_journal", go to next table.
Checking the next id sequence for table "tags".
nextval is NULL for table "tags", go to next table.
Checking the next id sequence for table "transaction_currencies".
Table "transaction_currencies" autoincrement is correct.
Checking the next id sequence for table "transaction_groups".
nextval is NULL for table "transaction_groups", go to next table.
Checking the next id sequence for table "transaction_journals".
nextval is NULL for table "transaction_journals", go to next table.
Checking the next id sequence for table "transaction_types".
Table "transaction_types" autoincrement is correct.
Checking the next id sequence for table "transactions".
nextval is NULL for table "transactions", go to next table.
Checking the next id sequence for table "users".
nextval is NULL for table "users", go to next table.
Checking the next id sequence for table "webhook_attempts".
nextval is NULL for table "webhook_attempts", go to next table.
Checking the next id sequence for table "webhook_messages".
nextval is NULL for table "webhook_messages", go to next table.
Checking the next id sequence for table "webhooks".
nextval is NULL for table "webhooks", go to next table.
Now decrypting the database (if necessary)...
Going to decrypt the database.
Decrypted the data in table "accounts".
Decrypted the data in table "attachments".
Decrypted the data in table "bills".
Decrypted the data in table "budgets".
Decrypted the data in table "categories".
Decrypted the data in table "piggy_banks".
Decrypted the data in table "preferences".
Decrypted the data in table "tags".
Decrypted the data in table "transaction_journals".
Decrypted the data in table "transactions".
Decrypted the data in table "journal_links".
Done!
Done!
Now executing firefly-iii:transaction-identifiers
All split journal transaction identifiers are correct.
Verified and fixed transaction identifiers in 0.01 seconds.
Now executing firefly-iii:migrate-to-groups
Found no split transaction journals. Nothing to do.
Migrate split journals to groups in 0 seconds.
No need to convert transaction journals.
Migrate all journals to groups in 0 seconds.
No journals to migrate to groups.
Now executing firefly-iii:account-currencies
All accounts are OK.
Verified and fixed account currencies in 0 seconds.
Now executing firefly-iii:transfer-currencies
All transfers have correct currency information.
Verified and fixed currency information for transfers in 0.01 seconds.
Now executing firefly-iii:other-currencies
Verified 0 transaction(s) and journal(s).
Verified and fixed transaction currencies in 0.01 seconds.
Now executing firefly-iii:migrate-notes
No notes to migrate.
Migrated notes in 0 seconds.
Now executing firefly-iii:migrate-attachments
All attachments are OK.
Migrated attachment notes in 0 seconds.
Now executing firefly-iii:bills-to-rules
All bills are OK.
Verified and fixed bills in 0 seconds.
Now executing firefly-iii:bl-currency
All budget limits are correct.
Verified budget limits in 0 seconds.
Now executing firefly-iii:cc-liabilities
No incorrectly stored credit card liabilities.
Now executing firefly-iii:back-to-journals
Check 0 transaction journal(s) for budget info.
Check 0 transaction journal(s) for category info.
Updated category and budget info for all transaction journals in 0.01 seconds.
Now executing firefly-iii:rename-account-meta
All account meta is OK.
Fixed account meta data in 0.01 seconds.
Now executing firefly-iii:migrate-recurrence-meta
No recurrence meta data migrated.
Migrated recurrence meta data in 0.01 seconds.
Now executing firefly-iii:migrate-tag-locations
Migrated tag locations in 0.01 seconds.
Now executing firefly-iii:migrate-recurrence-type
Update recurring transaction types in 0 seconds.
Now executing firefly-iii:upgrade-liabilities
Upgraded liabilities in 0.01 seconds.
Now executing firefly-iii:liabilities-600
Upgraded liabilities for 6.0.0 in 0 seconds.
Now executing firefly-iii:fix-piggies
All piggy bank events are correct.
Verified the content of 0 piggy bank events in 0 seconds.
Now executing firefly-iii:create-link-types
All link types OK!
Verified link types in 0 seconds
Now executing firefly-iii:create-access-tokens
All access tokens OK!
Verify access tokens in 0 seconds.
Now executing firefly-iii:remove-bills
All transaction journals have correct bill information.
Verified bills / journals in 0 seconds
Now executing firefly-iii:fix-negative-limits
All budget limits are OK.
Now executing firefly-iii:enable-currencies
0 different currencies are currently in use.
All currencies are correctly enabled or disabled.
Verified currencies in 0 seconds.
Now executing firefly-iii:fix-transfer-budgets
No invalid budget/journal entries.
Verified budget/journals in 0 seconds.
Now executing firefly-iii:fix-uneven-amount
Amount integrity OK!
Verified amount integrity in 0 seconds
Now executing firefly-iii:delete-zero-amount
No zero-amount transaction journals.
Verified zero-amount integrity in 0 seconds
Now executing firefly-iii:delete-orphaned-transactions
No orphaned journals.
No orphaned transactions.
No orphaned accounts.
Verified orphans in 0 seconds
Now executing firefly-iii:delete-empty-journals
No uneven transaction journals.
No empty transaction journals.
Verified empty journals in 0 seconds
Now executing firefly-iii:delete-empty-groups
Verified empty groups in 0 seconds
Now executing firefly-iii:fix-account-types
All account types are OK!
Verifying account types took 0 seconds
Now executing firefly-iii:fix-account-order
Verifying account order took 0 seconds
Now executing firefly-iii:rename-meta-fields
All meta fields are correct.
Renamed meta fields in 0.01 seconds
Now executing firefly-iii:fix-ob-currencies
Going to verify 0 opening balance transactions.
There was nothing to fix in the opening balance transactions.
Now executing firefly-iii:fix-long-descriptions
Verified all transaction group and journal title lengths.
Took 0 seconds.
Now executing firefly-iii:fix-recurring-transactions
Corrected recurring transactions in 0 seconds.
Now executing firefly-iii:unify-group-accounts
Updated inconsistent transaction groups.
Now executing firefly-iii:fix-transaction-types
All transaction journals are of the correct transaction type (in 0 seconds).
Now executing firefly-iii:fix-frontpage-accounts
Verifying account preferences took 0 seconds
Now executing firefly-iii:fix-ibans
Now executing firefly-iii:create-group-memberships
Validated group memberships in 0 seconds.
Now executing firefly-iii:upgrade-group-information
Now executing firefly-iii:report-empty-objects
Report on empty objects finished in 0.01 seconds
Now executing firefly-iii:report-sum
Report on total sum finished in 0 seconds
Now executing firefly-iii:restore-oauth-keys
Encryption keys generated successfully.
Now executing firefly:instructions update
+------------------------------------------------------------------------------+
|                                                                              |
| Thank you for updating to Firefly III, v6.0.7!                               |
|                                                                              |
|                                                                              |
+------------------------------------------------------------------------------+
Now executing firefly-iii:verify-security-alerts
```

**Comment by JC5** [comment]
Nice find. These tables are indeed no longer created (or used).
